ui/js/controllers/main.js

```diff
@@ -365,6 +365,12 @@ treeherderApp.controller('MainCtrl', [
             }
         };
 
+        $scope.getExcludedState = function() {
```

Comment:
I think this duplicates the functionality of the `$scope.isSkippingExclusionProfiles`, defined above. I think you should be able to just use that, instead of defining a new function.